core-web/apps/dotcms-ui/src/stories/primeng/form/AutoComplete.stories.ts

```diff
@@ -19,51 +43,95 @@ export default {
     },
     decorators: [
         moduleMetadata({
-            imports: [AutoCompleteModule, BrowserAnimationsModule]
+            imports: [AutoCompleteModule, NoopAnimationsModule, BrowserModule]
         })
     ],
     args: {
-        results: [
-            { name: 'Afghanistan', code: 'AF' },
-            { name: 'Albania', code: 'AL' },
-            { name: 'Venezuela', code: 'VE' }
-        ],
-        // tslint:disable-next-line: typedef
-        search() {
-            this.results = [
-                { name: 'Afghanistan', code: 'AF' },
-                { name: 'Albania', code: 'AL' },
-                { name: 'Venezuela', code: 'VE' }
-            ];
+        suggestions: [...originalData],
+        dropdown: true,
+        multiple: false,
+        field: 'name',
+        showClear: false,
+        dropdownIcon: 'pi pi-chevron-down',
```

Comment:
@fmontes, @zJaaal  What do you think about creating a global directive to set by default args like: `dropdownIcon`, `showClear`, `delay`, `appendTo` to no need to send those inputs every time you use the directive?

Reply:
At this point I feel is kinda overkill, since those are just inputs in the primeng components.